src/scripts/main.js

```diff
@@ -1,7 +1,23 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const notification = document.createElement('div');
+
+  notification.classList.add('notification', `${type}`);
```

Comment:
Template literal is redundant here as "type" is already a string and you don't add anything to it.